RiotNSE/RiotNSE.entitlements

```diff
@@ -6,8 +6,6 @@
 	<array>
 		<string>$(APPLICATION_GROUP_IDENTIFIER)</string>
 	</array>
-	<key>com.apple.developer.usernotifications.filtering</key>
```

Comment:
why did you remove this part ?

Reply:
Because, we currently don't have the feature in Tchap. And if this entitlement is set in the file, the app cannot build.
There is an issue to solve this [here](https://github.com/tchapgouv/tchap-ios/issues/635)